include/k4a/k4a.hpp

```diff
@@ -339,11 +348,11 @@ class image
 
     /** Set the image's timestamp in microseconds
      *
-     * \sa k4a_image_set_timestamp_usec
+     * \sa k4a_image_set_device_timestamp_usec
      */
     void set_timestamp(std::chrono::microseconds timestamp) noexcept
```

Comment:
do we want a set_device_timestamp and set_system_timestamp here now?

Reply:
yes, for the playback case